coap/idf_component.yml

```diff
@@ -1,4 +1,4 @@
-version: "4.3.1~3"
+version: "4.3.2~0"
```

Comment:
Any idea when the official 4.3.2 libcoap would be released? 

We can change this to `4.3.2~1-rc.1` till then.

Reply:
> Any idea when the official 4.3.2 libcoap would be released?

It should not be far off unless some major issues come up.

> We can change this to 4.3.2~1-rc.1 till then.

`4.3.2~0-rc.1` or `4.3.2~1-rc.1` ?

Reply:
Set to `4.3.2~1-rc.1`